lib/erb_lint/linters/deprecated_classes.rb

```diff
@@ -56,6 +60,10 @@ def violated_rules(class_name)
           /\A#{deprecated_rule[:class_expr]}\z/.match(class_name)
         end
       end
+
+      def pad_with_left_space(message)
+        message ? " #{message}" : ''
```

Comment:
I've moved the calls to `pad_with_left_space` out of the `initialize` method and down to `generate_errors` since it makes more sense to perform the calls there instead.